python/tvm/auto_scheduler/measure_record.py

```diff
@@ -61,8 +64,9 @@ class RecordReader(Object):
 
     def __init__(self, filename):
         # a set to prevent print duplicated message
+        if not os.path.exists(filename):
+            raise FileExistsError("%s does not exists!" % filename)
```

Comment:
- Move the comment in L66 to the right place.
- I'm thinking whether to error out or just a warning in this case. By looking into it use cases, it seems to me that a warning might be better, because some times users may write a script that preloads the record file whatever it exists or not.

Reply:
Modified.